src/wip/fixed-trader.rain

```diff
@@ -1,11 +1,11 @@
-# Strategy: Fixed Grid
+# Strategy: Fixed Trader
 #
 # High level, this strategy trades mean amount of BLUE denominated RED tokens at the end of every cooldown period
-# provided that current calculated BLUE denominated price of RED falls within the price gird.
+# provided that current calculated BLUE denominated price of RED falls above a set ratio.
```

Comment:
i don't think the defining quality of this strat is that it has a lower bound on the io ratio, that seems to just be a param you can set